slick/src/main/scala/slick/jdbc/OracleProfile.scala

```diff
@@ -324,13 +335,154 @@ trait OracleProfile extends JdbcProfile {
     }
 
     class UUIDJdbcType extends super.UUIDJdbcType {
+      override def sqlType = java.sql.Types.VARBINARY
       override def sqlTypeName(sym: Option[FieldSymbol]) = "RAW(32)"
       override def valueToSQLLiteral(value: UUID) = {
         val hex = value.toString.replace("-", "").toUpperCase
         s"hextoraw('$hex')"
       }
       override def hasLiteralForm = true
     }
+
+    class LocalDateJdbcType extends super.LocalDateJdbcType {
+      override def hasLiteralForm: Boolean = true
+      override def valueToSQLLiteral(value: LocalDate) : String = {
+        s"TO_DATE('${value.toString}', 'SYYYY-MM-DD')"
+      }
+      override def getValue(r: ResultSet, idx: Int): LocalDate = {
+        r.getString(idx) match {
+          case null => null
+          case dateStr => LocalDate.parse(dateStr.substring(0, 10))
+        }
+      }
+    }
+
+    class LocalTimeJdbcType extends super.LocalTimeJdbcType {
+      @inline private[this] def timestampFromLocalTime(localTime : LocalTime) : Timestamp = {
+        Timestamp.valueOf(LocalDateTime.of(LocalDate.MIN, localTime))
+      }
+      override def sqlType = java.sql.Types.TIMESTAMP
+      override def sqlTypeName(sym: Option[FieldSymbol]) = "TIMESTAMP(6)"
+
+      override def getValue(r: ResultSet, idx: Int) : LocalTime = {
+        r.getTimestamp(idx) match {
+          case null => null
+          case timestamp => timestamp.toLocalDateTime.toLocalTime
+        }
+      }
+    }
+
+    // LocalDateTime and Instant are the 2 types which have no TZ component
+    // So, store them at UTC timestamps, otherwise the JDBC layer might attempt to map them
+    // and with DST changes, there are some times which will be unrepresentable during the switchover
+    class LocalDateTimeJdbcType extends super.LocalDateTimeJdbcType {
+//      private[this] val formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSS x")
```

Comment:
remove the commented out code if there is no reason to keep it. If there is a reason, document it.

Reply:
Commented out code removed